packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -66,8 +75,81 @@ export default class HostItem extends Component {
   _toggleExpanded = () => this.setState({ expanded: !this.state.expanded })
   _onSelect = () => this.props.onSelect(this.props.item.id)
 
+  _getIcons = createSelector(
+    () => this.props.needsRestart,
+    () => this.props.item,
+    this._isMaintained,
+    (needsRestart, host, isMaintained) => {
+      const icons = []
+
+      if (needsRestart) {
+        icons.push({
+          level: 'warning',
+          render: (
+            <Link className='text-warning' to={`/hosts/${host.id}/patches`}>
+              <Icon icon='alarm' /> {_('rebootUpdateHostLabel')}
+            </Link>
+          ),
+        })
+      }
+
+      if (!isMaintained) {
+        icons.push({
+          level: 'warning',
+          render: (
+            <p>
+              <Icon icon='alarm' /> {_('noMoreMaintained')}
+            </p>
+          ),
+        })
+      }
+
+      if (!isHostTimeConsistentWithXoaTime(host)) {
+        icons.push({
+          level: 'danger',
+          render: (
+            <p>
+              <Icon icon='alarm' /> {_('warningHostTimeTooltip')}
+            </p>
+          ),
+        })
+      }
+
+      if (hasLicenseRestrictions(host)) {
+        icons.push({
+          level: 'danger',
+          render: (
```

Comment:
`render: <LicenseWarning />`  ?

Reply:
[LicenseWarning](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/host/license-warning.js#L26) will display an icon with a tooltip. This is not the behavior we want.

Reply:
You can adapt it for your use so that you don't have to rewrite the same code.

Reply:
[LicenseWarning](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/host/license-warning.js#L26) is also used [here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/host/tab-general.js). IMO we should avoid modifying this component.